sous-chef/sous_chef/recipe_book/_recipe_book.py

```diff
@@ -227,9 +240,41 @@ def _get_recipe_book() -> pd.DataFrame:
         )
         dataframe["factor"] = 1
         dataframe["amount"] = None
+        # cannot pickle pint units at this time
+        dataframe["quantity"] = None
         dataframe = dataframe.replace("nan", pd.NA)
+
+        dataframe.time_preparation = dataframe.time_preparation.astype(
```

Comment:
think these time conversions aren't needed anymore since we reverted back to only panderas.

Reply:
can handle later, added todo